editor.planx.uk/src/@planx/components/shared/Preview/OrNavigationButton.tsx

```diff
@@ -0,0 +1,77 @@
+import Box from "@mui/material/Box";
+import { styled } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import { ComponentType as TYPES } from "@opensystemslab/planx-core/types";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React from "react";
+import { ApplicationPath } from "types";
+
+import { contentFlowSpacing } from "./Card";
+import NavigateToPublishedButton from "./NavigateToPublishedButton";
+import SaveResumeButton from "./SaveResumeButton";
+
+type OrNavigationType = "save-resume" | "navigate-to-published";
+
+export const InnerContainer = styled(Box)(({ theme }) => ({
+  "& p": {
+    ...contentFlowSpacing(theme),
+  },
+}));
+
+const BUTTON_COMPONENTS = {
+  "save-resume": SaveResumeButton,
+  "navigate-to-published": NavigateToPublishedButton,
+} as const;
+
+const TEST_ENVIRONMENTS = new Set(["preview", "draft"]);
+
+const OrNavigationButton = ({
+  handleSubmit,
+}: {
+  handleSubmit: ((data?: any) => void) | undefined;
```

Comment:
```suggestion
  handleSubmit: ((data?: unknown) => void) | undefined;
```

We should actually come back and type `handleSubmit` properly, but for now let's work to not introduce any new `any`s to the codebase.